### PR TITLE
Replace simple quote by space in legend

### PR DESCRIPTION
**Pull request by s-duret**
Replace hash by hash isn't useful
I replace simple quote by space because graph wasn't displayed with this character

### CODE REVIEW

File: www/class/centreonGraph.class.php [hunk 1]
```diff
@@ -282,7 +282,7 @@ protected function cleanupDsName($dsname, $reverse = false)
     */
     protected function cleanupDsNameForLegend($dsname, $reverse = false)
     {
-        $newDsName = str_replace(array("slash_", "bslash_", "pct_",  "#", "\\"), array("/", "\\", "%", "#", "\\\\"), $dsname);
```
Comment (julienmathis) [inline www/class/centreonGraph.class.php]: Why did you remove the line 285 ?

Comment (s-duret) [inline www/class/centreonGraph.class.php]: Because I think replacing # by # isn't useful.

Comment (julienmathis) [inline www/class/centreonGraph.class.php]: Are you sure ? :)
